### DISCUSSION TITLE
Welcome to EasyMocap Discussions!

### DISCUSSION THREAD

**Discussion by chingswy** [zju3dv/EasyMocap#177] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gqbay** [comment]
Hello everyone! I'm Bay. Digital artist from Kyrgyzstan. 

 I think I follow this project since it was revealed on GitHub. Huge thanks for the devs for such a great opportunity!  Also thanks for @carlosedubarreto for guiding me, I couldn't test easymocap w/o his help!

My biggest wish/idea/desire I'm dreaming about is if easymocap could also estimate the shape, so the person from the footage and the final animated mesh would be matching, not like SMPL model over person. Even roughly  

Example of technology by Juergen Gall and team from 2009 or 2012 if I'm not mistaken:
https://user-images.githubusercontent.com/65506709/181085975-448d8aa7-7630-4624-aef1-de44a0fb9da3.mp4

And questions:
1. No doubts everyone now waiting for updated version of easymocap that will allow to use video from internet. So the question is, is there any approximate date? As far as I can see you are already updating https://github.com/chingswy/easymocap-public-doc at least a few times* a week, which makes me think* it will be updated really soon!
2. speaking of that updates, I saw some demo videos in /[works](https://github.com/chingswy/easymocap-public-doc/tree/main/works)/multinb/ They are fantastic and made me think that my biggest wish/idea/desire that I started with will come true but even better than I could imagine?  Not sure, but anyways, demos looks splendid! 

That's everything I wanted to start with.
Warmest regards!
